lib/evaluation/handlers/subjectIsHolderEvaluationHandler.ts

```diff
@@ -211,20 +206,23 @@ export class SubjectIsHolderEvaluationHandler extends AbstractEvaluationHandler
 
   private confirmFieldSetHasSameHolder(status: 'info' | 'warn' | 'error') {
     return (inputDescriptorIds: Set<string>, fieldIdSet: Set<string>) => {
-      const credentialSubjectsSet: Set<unknown> = new Set<unknown>();
-      inputDescriptorIds.forEach((inDescId) => {
+      const credentialSubjectsSet: Set<JsonpathType> = new Set<JsonpathType>();
```

Comment:
Name implies it is something different from a path

Reply:
Yes! It's the jsonpath + the actual credentialSubject object. Fixed